packages/syntax-icons/src/Icons.stories.tsx

```diff
@@ -0,0 +1,370 @@
+];
+
+const MUIIcons = [{ name: "Repeat", component: Repeat }];
+
+export default {
+  title: "Icons/Icons",
+  component: Icon,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/%F0%9F%93%90-Syntax?node-id=1206-4420&t=yFh7Ijhf6PU7Lin3-0",
+    },
+  },
+  argTypes: {
+    size: {
+      options: ["xs", "sm", "md", "lg"],
+      control: { type: "radio" },
+    },
+    color: {
+      options: [
+        "black",
+        "gray700",
+        "gray900",
+        "primary",
+        "destructive-primary",
+        "destructive-darkBackground",
+        "success",
+        "success-darkBackground",
+        "white",
+        "inherit",
+      ],
+      control: { type: "radio" },
+    },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof Icon>;
+
+const getBackgroundColor = (
+  color:
+    | "black"
+    | "gray900"
+    | "gray700"
+    | "primary"
+    | "destructive-primary"
+    | "destructive-darkBackground"
+    | "success"
+    | "success-darkBackground"
+    | "white"
+    | "inherit"
+    | undefined,
+) => {
+  const listOfLightFontColors = [
+    "white",
+    "destructive-darkBackground",
+    "success-darkBackground",
+  ];
+
+  if (!color) return "white";
+
+  if (listOfLightFontColors.includes(color)) {
+    return "black";
+  }
+
+  return "white";
+};
+
+export const Default: StoryObj<typeof Icon> = {
+  args: {
+    size: "lg",
+    color: "primary",
+  },
+  render: ({ size, color }) => {
+    return (
+      <Box display="flex" flexWrap="wrap" gap={4}>
+        {cambioIcons.map((icon) => {
+          const IndIcon = icon.component;
+          return (
+            <Box
+              key={icon.name}
+              display="flex"
+              direction="column"
+              gap={2}
+              justifyContent="center"
+              alignItems="center"
+              maxWidth={150}
+              width="100%"
+              padding={2}
+              backgroundColor={getBackgroundColor(color)}
+            >
+              <IndIcon size={size} color={color} />
+              <Typography color={color} size={100}>
+                {icon.name}
+              </Typography>
+            </Box>
+          );
+        })}
+        {MUIIcons.map((icon) => {
+          const IndIcon = icon.component;
+          return (
+            <Box
+              key={icon.name}
+              display="flex"
+              direction="column"
+              gap={2}
+              justifyContent="center"
+              alignItems="center"
+              maxWidth={150}
+              width="100%"
+              padding={2}
+              backgroundColor={getBackgroundColor(color)}
+            >
+              <IndIcon />
+              <Typography color={color} size={100}>
+                {icon.name}
+              </Typography>
+            </Box>
+          );
+        })}
+      </Box>
+    );
+  },
+};
+
+export const OfficialCambio: StoryObj<typeof Icon> = {
```

Comment:
@somethiiing I would remove `OfficialCambio` and `MUIIconsToBeMigrated` and only include official Cambio icons